### PR TITLE
Hidden option

### PR DESCRIPTION
**Pull request by nadar**
+ hide hidden pages by default
+ add configuration option for module to include hidden pages into the sitemap.
+ added basic unit test

Todo:
+ unit test does not return the stream content, currently i dunno why the response object stream can not be parsed to a content string in order to compare the result.
+ The require-dev dev-master for cms and admin module can be removed after next release.

### CODE REVIEW

File: tests/SitemapControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        $module->withHidden = false;
+        
+        $this->prepareBasicTableStructureAndData();
+        
+        $ctrl = new SitemapController('sitemap', $module);
+        $response = $ctrl->actionIndex();
+        $stream = $response->stream;
+        
+        // @TODO: Does not deliver the xml stream content ...
+        $content = stream_get_contents($stream[0]);
```
Comment (nadar) [inline tests/SitemapControllerTest.php:51]: @cebe Do you know why the yii\web\Response stream can not be converted into a string in oder to compare?

File: src/Module.php [hunk 2]
```diff
@@ -11,5 +11,9 @@
  */
 final class Module extends \luya\base\Module implements CoreModuleInterface
 {
-
+    /**
+     * @var boolean Whether hidden page should be included or not, defaults to false in order to make sure only
+     * visible pages are included by default. In order to include hidden pages into the sitemap enable $withHidden.
+     */
+    public $withHidden = false;
```
Comment (nadar) [inline src/Module.php:18]: I was not sure how to name the property, other options could be:

+ withHidden
+ ignoreHidden
+ showHidden
+ includeHidden

**Comment by nadar** [comment]
@cebe any news on this?

File: src/Module.php (hunk 2 above)
Comment (cebe) [reply src/Module.php:18]: `$withHidden` sounds fine to me

**Comment by cebe** [comment]
Merged, thank you!

**Pull request closed without merging**